.github/workflows/test.yml

```
@@ -72,3 +72,4 @@ jobs:
         env: 
           TOXENV: ${{ matrix.toxenv }}
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}"
```

I think there's an errant `"` here that's causing the 500 from Coveralls.
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

---

Once that's fixed, this PR looks good to me! I'll get it merged and a new version released.

---

> I think there's an errant `"` here that's causing the 500 from Coveralls.
> ```suggestion
>           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
> ```
> 

🤦‍♂️

Thanks for pointing that out!